src/localAnsibleRunner.ts

```diff
@@ -31,7 +31,21 @@ export class LocalAnsibleRunner extends TerminalBaseRunner {
         if (utilities.isTelemetryEnabled()) {
             cmdsToTerminal.push(envCmd + Constants.UserAgentName + '=' + utilities.getUserAgent());
         }
-        cmdsToTerminal.push(this.getRunPlaybookCmd("\"" + playbook + "\""));
+
+        if (utilities.isWslEnabled()) {
+            var sourcePath = path.dirname(playbook);
+            var targetPath = '/playbook';
```

Comment:
why the targetpath is hardcoded?

Reply:
This was based on the existing path translation that is present in the dockerRunner.ts.

Reply:
shouldn't it be sth like /mnt/c/….  in case of wsl?
just need to check how it works.

Reply:
For docker scenario, extension will mount local workspace as`playbook` in docker container. in wsl, it should be same folder name without any change, so not `playbook`.

Reply:
I've clean up the playbook path translation

Reply:
Just realized a usecase that needs to be addressed either in a future one PR ... playbook or workspace residing on a non-local drive or network path.